dev/com.ibm.websphere.appserver.features/visibility/public/batch-2.0/resources/l10n/io.openliberty.batch-2.0_cs.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
```

Comment:
Copyright statements in new feature sources should indicate 2020, only.  Please review the change set and revise the copyrights dates where appropriate.